.eslintignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-dist
\ No newline at end of file
+dist
+build
+global
```

Comment:
The cause if the pipeline failures. This is where our exported files are placed.